src/chi/components/dropdown/dropdown.scss

```diff
@@ -0,0 +1,211 @@
+@import 'variables';
+@import 'mixins';
+
+.chi {
+
+  .m-dropdown {
+    display: inline-block;
+    position: relative;
+
+    .m-dropdown__menu {
+      left: 0;
+      min-width: 100%;
+      position:absolute;
+      z-index: 1;
+    }
+
+    &__trigger {
```

Comment:
@dani-cl-madrid @mattnickles I would remove the trigger in the scss file and in the documentation page as it will enter in the JS part with the functionality.

Reply:
@jllr It's needed to render the icon in the examples. Without it, all we'd be showing are the menus